src/board/index.ts

```diff
@@ -425,8 +473,7 @@ export class Board {
    * reset() in MicroPython code throws ResetError.
    */
   async reset(): Promise<void> {
-    const noChangeRestart = () => {};
-    this.stop(noChangeRestart);
+    this.stop(StopKind.Reset);
```

Comment:
Not a new bug, but it feels like this should return the promise.

Somewhat academic as we'll do the restart by calling start in setTimeout but at least it wouldn't look broken.

Reply:
c8c4d44bbd20bac4b496d68bcca7bf8a4859b486